README.md

```diff
@@ -311,6 +311,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Lint pull request name with commitlint (Awesome if you squash merge !)](https://github.com/JulienKode/pull-request-name-linter-action)
 - [Block PR merges when Checks for target branches are failing](https://github.com/cirrus-actions/branch-guard)
 - [Get generated static site screeshots updated by Pull Request](https://github.com/ssowonny/diff-pages-action)
+- [A tool that can be used to enforce policies on git repositories, including conventional commits](https://github.com/talos-systems/conform)
```

Comment:
I would move this to the "GitHub Tools and Management" section. Also, please rename the title to "Enforce Policies on GitHub Repositories and Commits". For future reference, please follow Title Casing guidelines found here: https://github.com/sdras/awesome-actions/blob/master/contributing.md